step1-clean/db.py

```diff
@@ -0,0 +1,25 @@
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import create_engine, Column, String, Integer
+
+Base = declarative_base()
+
+class SQLImage(Base):
+  __tablename__ = 'Thumbnails'
+
+  id = Column(Integer, primary_key=True)
+  filename = Column(String)
+  isThumbnail = Column(String)
```

Comment:
Also, you for flags you'll want to use `Boolean` from sqlalchemy instead of string